src/plugins/kibana_react/public/icon/field_icon.tsx

```diff
@@ -24,7 +24,7 @@ interface IconMapEntry {
   icon: string;
   color: string;
 }
-interface FieldNameIconProps {
+interface FieldIconProps {
```

Comment:
can we also call the folder `field_icon` together with the component? I'm sure it's not done consistently across the app but if the folder mainly exports a single component it should be called like the component